Monika After Story/game/script-affection.rpy

```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
 
         ## DISTRESSED quips
         quips = [
+            _("..."),
             _("...Yes?"),
-            _("...Oh?"),
-            _("...Huh?"),
             _("...Hm?"),
-            _("We can try talking, I guess."),
-            _("I guess we can talk."),
-            _("Oh... You want to talk?"),
-            _("If you want to talk, go ahead."),
-            _("We can talk if you really want to."),
+            _("...Hello?"),
+            _("Oh..."),
+            _("Huh..."),
+            _("...we can talk."),
+            _("...go ahead."),
+            _("...okay."),
             _("Are you sure you want to talk to me?"),
             _("You actually want to talk to me?"),
-            _("Alright...if you want to talk with me."),
-            _("You sure you want to talk?")
+            _("Alright...if that's what you want."),
         ]
         save_quips(DISTRESSED, quips)
 
         ## UPSET quips
         quips = [
+            _("..."),
             _("What?"),
+            _("Huh?"),
+            _("Yeah?"),
             _("What do you want?"),
             _("What now?"),
             _("What is it?"),
-#            "Fine...we can talk.",
-#            "Just...whatever, go ahead."
+            _("Go on then."),
+            _("Let's hear it."),
+            _("Is this really what you want?"),
+            _("Don't make fun of me."),
+            _("This better be important."),
```

Comment:
To match with the above, we can slightly modify this to be a little less aggressive
```suggestion
            _("I hope this is important."),
```